CHANGELOG.rst

```diff
@@ -18,6 +18,76 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 4.6.0 (2019-05-31)
+=========================
+
```

Comment:
We should make it very clear that this is the last major series to support Python 2. I suggest:

```rst
Important
---------

The ``4.6.X`` series will be the last series to support **Python 2 and Python 3.4**. 

For more details, see our `Python 2.7 and 3.4 support plan <https://docs.pytest.org/en/latest/py27-py34-deprecation.html>`__.
```